config/routes.rb

```diff
@@ -4,6 +4,7 @@
   namespace :api do
     post 'login', to: 'users#login'
     get 'feed', to: 'scores#user_feed'
-    resources :scores, only: %i[create destroy]
```

Comment:
I see you added the `create` and `destroy` methods back in the `scores_controller`, but you also need to add the routing back, otherwise the actions are not reachable. Add `resources :scores, only: %i[create destroy]` back in `routes.rb`.